tron/mesos.py

```diff
@@ -164,10 +165,14 @@ def __init__(self, id, task_config, serializer=None):
         self.log = self.get_event_logger()
         self.setup_output_logging()
 
+        config_str = str(self.get_config())
+        # AWS_SECRET_ACCESS_KEYs are Base64-encoded
```

Comment:
Out of curiosity, how does the base64 encoding affect this code? should the test have base64 encoding too?

Reply:
It just affects the regex.  Lemme make that comment a bit clearer.